probdists/Gammadistribution.py

```diff
@@ -102,6 +102,34 @@ def plot_bar_pdf(self, points=25):
         plt.show()
         return x, y
 
+    def cdf_lower(self, x):
+        """
+        Cumulative density function calculator for the Gamma distribution.
+            Args:
+                x (float): point for calculating the cumulative distribution function
+            Returns:
+                float: lower cumulative distribution function output (1-cdf)
+        """
+        #initiate cdf variable
+        cdf = 0
+        for i in range (self.k):
+            cdf += (math.pow((x / self.theta), i) * math.exp(-1 * x / self.theta)) / math.factorial(i) 
+        return 1-cdf
+
+    def cdf_upper(self, x):
+        """
+        Cumulative density function calculator for the Gamma distribution.
+            Args:
+                x (float): point for calculating the cumulative distribution function
+            Returns:
+                float: upper cumulative distribution function output (cdf)
+        """
+        #initiate cdf variable
+        cdf = 0
+        for i in range (self.k):
+            cdf += (math.pow((x / self.theta), i) * math.exp(-1 * x / self.theta)) / math.factorial(i) 
+        return cdf
+
```

Comment:
Could you provide a reference for this? I wasn't able to find much information on an upper cdf for gamma distribution on wikipedia.

Reply:
i think this isn't a good way to do get cdf value. You wrote the same code twice their only change is
 
in cdf_lower returned  1 - cdf 
and cdf_upper returend the cdf

you can make an argument to set if user asking the lowe or upper. So the user can get cdf with one function. And also you reduce the redundancy.